cockpit/src/subscriptions.css

```diff
@@ -17,8 +17,6 @@
  * along with Cockpit; If not, see <http://www.gnu.org/licenses/>.
  */
 
-@import '~patternfly/dist/css/patternfly-additions.min.css';
-
 .installed-products .list-view-pf-left .fa.list-view-pf-icon-sm {
     border: none;
```

Comment:
The green icon is not vertically aligned with other elements as you can see here:

![obrazek](https://user-images.githubusercontent.com/2057012/86016164-bfef3200-ba22-11ea-8f23-b1132901a015.png)

I would add here following to fix this issue:
```css
margin-top: 4px;
```